libexec/rtld-elf/rtld_c18n.c

```diff
@@ -1587,8 +1590,14 @@ c18n_init2(void)
 	expand_tramp_table(9);
 
 	/*
-	 * Create the first trampoline page.
+	 * Find a suitable page size and create the first trampoline page.
 	 */
+	while (n-- > 0)
```

Comment:
```
for (n = npagesizes - 1; n >= 0; --n) {
    ...
}
```
please (and you can then remove the initialiser at the start of the function), as a simpler / more normal style